cmd/kubelet/app/server.go

```diff
@@ -221,6 +221,12 @@ is checked every 20 seconds (also configurable with a flag).`,
 				}
 			}
 
+			if kubeletFlags.DisableCgroupV1Support {
+				if !isCgroup2UnifiedMode() {
+					return fmt.Errorf("Kubelet is not permitted to run on a host using cgroup v1")
```

Comment:
I think, if we were to make this an error, then...

```suggestion
return fmt.Errorf("kubelet is not permitted to run on a host using cgroup v1")
```

But, I also wonder whether a hard stop wouldn't be better here? Perhaps...

```suggestion
klog.Fatal(Kubelet is not permitted to run on a host using cgroup v1")
```

Reply:
@kwilczynski every other option when failed during that validation returns an error (which eventually results in kubelet failing to start).

Reply:
The "permitted" might also be replaced with "allowed" or "enabled". Where "permitted" is the most formal, and "allowed" is most frequently used within the code base.

Reply:
Thanks.